code/workspaces/web-app/src/containers/adminListUsers/AdminUsersContainer.spec.js

```diff
@@ -16,10 +16,12 @@ jest.mock('../../hooks/stacksHooks');
 jest.mock('../../hooks/dataStorageHooks');
 jest.mock('../../hooks/projectsHooks');
 jest.mock('../../config/catalogue');
-jest.mock('./UserResources', () => ({
-  __esModule: true,
-  default: jest.fn().mockReturnValue(<>user resources</>),
-}));
+// jest.mock('./UserResources', () => ({
```

Comment:
Can you get rid of this now if the below mock works?

Reply:
doh, missed deleting that - done now